requirements-debian.txt

```diff
@@ -4,8 +4,6 @@
 uWSGI==2.0.16
 
 # These have pre-compiled manylinux packages on CPython but not on PyPy
-cffi==1.11.4; platform_python_implementation!='CPython'
 cryptography==2.1.4; platform_python_implementation!='CPython'
 Pillow==5.0.0; platform_python_implementation!='CPython'
-# pyscopg2 is not compatible with PyPy
-lxml==4.1.1; platform_python_implementation!='CPython'
\ No newline at end of file
+lxml==4.1.1; platform_python_implementation!='CPython'
```

Comment:
Did the `lxml` line change at all? Maybe a newline added/removed?

Reply:
Trailing newline added.